### PR TITLE
add food update functionality

### PR DESCRIPTION
**Pull request by cmacaulay**
🎉 🎉 🗡 💃 
- User can update a food's calories
- User can update a food's name
- Changes are updated in the dom upon click away
- Changes persist across refreshing 
👎 
- Still need to implement a binding to user hitting enter for same functionality as click away
❓ 
@case-eee is it ok that upon refresh, the foods are reordered with the most recently updated food on top?

### CODE REVIEW

File: lib/foods.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ $(document).ready(function() {
   $('.food-options').on('click', '.food-delete', () => {
     handleDeletingFood();
   });
-
+  $('.food-options').on('click', '.food-name, .food-calories', () => {
```
Comment (cmacaulay) [inline lib/foods.js:11]: 🥇 @case-eee & @TheKaterTot there are two small pieces of functionality missing here, and I would appreciate any thoughts/suggestions either of you have in order to achieve them:
1) On a click, it creates an input field for the user to update either a name or calorie for a food. However, the user has to click into the field in order to start typing. Is there a way to automatically place a cursor into the input field?
2) The new value saves to the DOM and is updated in the DB when the user clicks out of field -- using enter to submit does not yet work, is the best way to approach it to create another event listener for an enter key press? 

Thank you! 🙋

Comment (cmacaulay) [reply lib/foods.js:11]: @lfollansbee thanks for taking a look!

Comment (TheKaterTot) [reply lib/foods.js:11]: I thought that pushing the 'enter' key worked automatically in forms. So maybe if the input field was wrapped in a form? I've seen plenty of examples of forms where they had only one or two input fields.

File: lib/foods.js [hunk 2]
```diff
@@ -36,7 +38,7 @@ function populateTable () {
 
 function addRow(food) {
   let foodRow = `
-      <tr class='food'>
+      <tr class='food' id=${food.id}>
```
Comment (TheKaterTot) [inline lib/foods.js:41]: Good job adding this unique selector!

**Comment by TheKaterTot** [comment]
Really good work on this! This is a tricky feature, and you worked it out ⚙️ ⚙️

**Comment by case-eee** [comment]
@cmacaulay thanks for asking your question in slack as well 👍 it's okay if they are reordered!

**Pull request merged**